src/communication/hpxreceiver.h

```diff
@@ -62,6 +63,20 @@ class HPXReceiver : public hpx::components::simple_component_base<HPXReceiver<CA
         return ids;
     }
 
+    virtual ~HPXReceiver()
```

Comment:
Ups, I played around fixing the issues I ran into. Nothing changed there, I merely forgot to undo those changes.